expyfun/_experiment_controller.py

```diff
@@ -917,7 +918,8 @@ def listen_presses(self):
         """
         self._response_handler.listen_presses()
 
-    def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
+    def get_presses(self, live_keys=None, timestamp=True, relative_to=None,
+                    releases=False):
```

Comment:
The behavior that we want is that every time we call `get_presses` the entire pyglet / cedrus buffer gets cleared.  The docstring will need to make it very clear (possibly in a Notes section) that if you call `get_presses(..., type='releases')` then the press information will be discarded.  (similarly for `get_presses(..., type='presses')` then the releases will be discarded).  The reason for doing it this way is that the alternative is worse: if users only care about presses for a while, then at some point they start caring about releases, the first time they get releases there will be a big backlog of them from all the earlier unretrieved release events.